### ISSUE TITLE
Experience enhancement of paddlenlp.transformers module

### ISSUE THREAD

**Issue by yingyibiao** [PaddlePaddle/PaddleNLP#2356]
- [ ] 完善 attention_mask 模块

- https://github.com/PaddlePaddle/PaddleNLP/issues/1850
- https://github.com/PaddlePaddle/PaddleNLP/pull/2005

- [ ] 完善 model_input_names 字段

- [ ] 模型下游任务支持传入 labels 并输出 loss

- [ ] 返回 hidden_states，attentions 等模型内部信息

- https://github.com/PaddlePaddle/PaddleNLP/pull/2583

- [x] 支持词表扩充

- https://github.com/PaddlePaddle/PaddleNLP/pull/2423

- [ ] 支持输入序列最大长度扩充

- https://github.com/PaddlePaddle/PaddleNLP/issues/2490
- https://github.com/PaddlePaddle/PaddleNLP/pull/2513

- [ ] 支持模型接收 inputs_embeds 字段

- https://github.com/PaddlePaddle/PaddleNLP/issues/2580

- [ ] 支持修改 word_embeddings

- [x] `from_pretrianed` 加载内置模型时缓存config文件，保证离线时可以使用缓存目录 @wj-Mcat 

- [ ] `from_pretrianed` 可以只使用 config 文件创建模型并使用随机初始化 

- [ ] config 获取以及如何从config创建模型

**Comment by wj-Mcat** [comment]
@guoshengCS 上面列表的内容我们都是在部分模型（并非是所有的模型）上面完成了，而且已经有其他任务安排了，所以这个 issue 可暂时关掉。
